2021-05-12 19:47:55,582 INFO    MainThread:17096 [wandb_setup.py:_flush():69] setting env: {}
2021-05-12 19:47:55,582 INFO    MainThread:17096 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-12 19:47:55,582 INFO    MainThread:17096 [wandb_init.py:_log_setup():336] Logging user logs to C:\Users\KukJinKim\Google 드라이브\김국진\대학\4-1\소프트웨어 융합 캡스톤 디자인\ApexDQFD\wandb\run-20210512_194755-3qfv8903\logs\debug.log
2021-05-12 19:47:55,582 INFO    MainThread:17096 [wandb_init.py:_log_setup():337] Logging internal logs to C:\Users\KukJinKim\Google 드라이브\김국진\대학\4-1\소프트웨어 융합 캡스톤 디자인\ApexDQFD\wandb\run-20210512_194755-3qfv8903\logs\debug-internal.log
2021-05-12 19:47:55,583 INFO    MainThread:17096 [wandb_init.py:_jupyter_setup():287] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000194FEEA6288>
2021-05-12 19:47:55,583 INFO    MainThread:17096 [wandb_init.py:init():369] calling init triggers
2021-05-12 19:47:55,583 INFO    MainThread:17096 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-12 19:47:55,583 INFO    MainThread:17096 [wandb_init.py:init():418] starting backend
2021-05-12 19:47:55,583 INFO    MainThread:17096 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=spawn, using: spawn
2021-05-12 19:47:55,589 INFO    MainThread:17096 [backend.py:ensure_launched():123] starting backend process...
2021-05-12 19:47:55,686 INFO    MainThread:17096 [backend.py:ensure_launched():128] started backend process with pid: 64492
2021-05-12 19:47:55,687 INFO    MainThread:17096 [wandb_init.py:init():423] backend started and connected
2021-05-12 19:47:55,701 INFO    MainThread:17096 [wandb_init.py:init():463] updated telemetry
2021-05-12 19:47:55,702 INFO    MainThread:17096 [wandb_init.py:init():482] communicating current version
2021-05-12 19:47:57,316 INFO    MainThread:17096 [wandb_init.py:init():487] got version response upgrade_message: "wandb version 0.10.30 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-12 19:47:57,317 INFO    MainThread:17096 [wandb_init.py:init():495] communicating run to backend with 30 second timeout
2021-05-12 19:47:57,953 INFO    MainThread:17096 [wandb_init.py:init():520] starting run threads in backend
2021-05-12 19:48:01,073 INFO    MainThread:17096 [wandb_run.py:_console_start():1507] atexit reg
2021-05-12 19:48:01,074 INFO    MainThread:17096 [wandb_run.py:_redirect():1370] redirect: SettingsConsole.WRAP
2021-05-12 19:48:01,074 INFO    MainThread:17096 [wandb_run.py:_redirect():1405] Wrapping output streams.
2021-05-12 19:48:01,075 INFO    MainThread:17096 [wandb_run.py:_redirect():1421] Redirects installed.
2021-05-12 19:48:01,075 INFO    MainThread:17096 [wandb_init.py:init():544] run started, returning control to user process
2021-05-12 19:48:01,079 INFO    MainThread:17096 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-12 19:48:01,079 INFO    MainThread:17096 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-12 19:48:01,083 INFO    MainThread:17096 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-12 19:48:01,084 INFO    MainThread:17096 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-12 19:48:01,084 INFO    MainThread:17096 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-12 19:48:01,090 INFO    MainThread:17096 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-12 19:48:01,092 INFO    MainThread:17096 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-12 19:48:01,092 INFO    MainThread:17096 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-12 19:48:01,106 INFO    MainThread:17096 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-12 19:48:01,110 INFO    MainThread:17096 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-12 19:48:01,110 INFO    MainThread:17096 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-12 19:48:01,116 INFO    MainThread:17096 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-12 19:48:01,130 INFO    MainThread:17096 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-12 19:48:01,130 INFO    MainThread:17096 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-12 19:48:01,150 INFO    MainThread:17096 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-12 19:48:01,154 INFO    MainThread:17096 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-12 19:48:01,154 INFO    MainThread:17096 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-12 19:48:01,162 INFO    MainThread:17096 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-12 19:48:01,172 INFO    MainThread:17096 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-12 19:48:01,172 INFO    MainThread:17096 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-12 19:48:01,178 INFO    MainThread:17096 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-12 19:48:02,965 INFO    MainThread:17096 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-12 19:48:02,965 INFO    MainThread:17096 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-12 19:48:05,469 INFO    MainThread:17096 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-12 19:48:09,703 INFO    MainThread:17096 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-12 19:48:09,703 INFO    MainThread:17096 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-12 19:48:19,569 INFO    MainThread:17096 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-12 19:52:48,988 INFO    MainThread:17096 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-12 19:52:48,989 INFO    MainThread:17096 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-12 19:52:51,329 INFO    MainThread:17096 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-12 19:53:27,909 INFO    MainThread:17096 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-12 19:53:27,909 INFO    MainThread:17096 [jupyter.py:save_ipynb():265] not saving jupyter notebook
